core/src/test/scala/stryker4s/mutants/applymutants/MatchBuilderTest.scala

```diff
@@ -79,21 +79,21 @@ class MatchBuilderTest extends Stryker4sSuite with TreeEquality {
       // Assert
       val expected =
         """class Foo {
-          |  def bar: Boolean = (sys.env.get("ACTIVE_MUTATION") match {
+          |  def bar: Boolean = sys.env.get("ACTIVE_MUTATION") match {
           |    case Some("0") =>
-          |      15 < 14
+          |      15 < 14 && 14 >= 13
```

Comment:
Am missing a mutation here:

`case GreaterThan(orig)        => orig ~~> (GreaterThanEqualTo, LesserThan, EqualTo)` 
Where is the `GreaterThanEqualTo` ?

Reply:
The MutantMatcher wasn't being used here, so the mutations were added above. I've changed it to three to reflect the amount that'd show up in a real case, though